app/src/util/types/fhir.ts

```diff
@@ -1,8 +1,10 @@
 export interface CQFMMeasure extends fhir4.Measure {
+  id: string;
```

Comment:
It's a bit confusing in the spec actually because I agree that `id` is not required for the CQFM profiles; however, it seems as though `id` is not just required for our implementation. There are multiple places in the spec that seem to insinuate that a server-defined `id` is required. There are some spots:
- _logical id_: https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#artifact-identity
- SHOULD params for requriements: https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#requirements
- SHOULD params for package: https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#package

Am I interpreting this correctly? Happy to add a comment if you think that would be helpful. If we remove it from the interface, the typing some places gets tricky because id will be `string | undefined` but we know it must be defined.